YAML typeset changes figures/objects

alexhallenbeck opened the pull request:
@geealbers I've made all the changes I knew how to do, but some questions for VEs or you remain in red text after my initials on the spreadsheet. I deleted the objects and/or figures requested, and created an issue #30 with a list of them. 

If there's anything on the spreadsheet I should have been able to do--or could pretty easily learn how to do--please let me know.

alexhallenbeck committed:
update object info batch 1

alexhallenbeck committed:
typeset changes batch 2

alexhallenbeck committed:
.yaml typeset changes batch 3

alexhallenbeck committed:
yaml typeset changes batch 4

geealbers commented on content/_data/objects.yaml:
```diff
@@ -601,23 +601,23 @@ object_list:
-    featured: 
-    type: [ press clipping ]
-    figure: [ id: "fig-035" ]
+# -
```
When removing an object from the project entirely (versus removing a single figure from an object) you'll also need to delete the corresponding Markdown file in the `content/object-index/` directory, and any references to the object in the main chapter Markdown files. In this case, there's a reference to object 035 in the list of press clippings in `content/01/index.md` (the Feldman chapter).

geealbers commented on content/_data/objects.yaml:
```diff
@@ -4229,27 +4229,27 @@ object_list:
-    location: "Getty Center, Los Angeles"
-    extended_caption: |
-      The [score for Earle Brown’s *December 1952*](/object-index/301/) is little more than a series of horizontally and vertically oriented black rectangles spread out on a square piece of paper. One of the earliest instances of graphic notation, *December 1952* is largely open to the choices of the performer: the page may be oriented in any direction, while the shapes represent sound events in  ...
+# -
```
See note above regarding object removal

geealbers reviewed (changes requested):
Thanks @alexhallenbeck. Please see my notes about extra steps removing objects from the project entirely. And I assume you've already run this through a YAML validator, but please do so if not.

alexhallenbeck committed:
deleted object md

alexhallenbeck commented:
@geealbers thanks for letting me know. I deleted the individual markdown pages for these two objects and deleted the references to them in the index (each had one). Let me know if this look OK!

geealbers reviewed (approved):
Perfect, thanks!
